### ISSUE TITLE
A11y_api-catalog_System.Activities.Debugger_Keyboard: User is not able to access "%apps" control present in graph using keyboard

### ISSUE THREAD

**Issue by Amulya-95** [terrajobst/apisof.net#16]
[Check out Accessibility Insights](https://www.1eswiki.com/wiki/Accessibility_Insights) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**
#A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_api-catalog; #MAS2.1.1; #Keyboard; #C+E-H; #api-catalog; #Edge; #Chrome; #A11ySev2; #NonBenchmark;

**Environment Details:**
URL: https://apisof.net/

**Browser:**
Microsoft Edge 44.18362.267.0
Chrome Version 76.0.3809.100

**Repro steps:**
1)Hit the URL : "http://apisof.net" and press enter
2)Tab till "Browse" button and press enter
3)Tab till "System.Activities.Debugger" and press enter
4)Check whether user is able to access "%apps" control present in graph using keyboard

**Actual Result:**
User is not able to access "%apps" control present in graph using keyboard

**Expected Result:**
User should be able to access "%apps" control present in graph using keyboard.
If the controls are accessible using mouse then keyboard only Users should be able to access control present in graph using keyboard

**User Impact:**
Keyboard only users will find difficulty if user is not able to access control using keyboard

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

[Not able to access %app control.pptx](https://github.com/terrajobst/apisof.net/files/3533630/Not.able.to.access.app.control.pptx)

**Comment by terrajobst** [comment]
The web site no longer exists.
